integrations/cms-googlesheets/src/GoogleSheetsCms.ts

```diff
@@ -0,0 +1,103 @@
+import { App, Jovo, JovoError, Plugin, PluginConfig } from '@jovotech/framework';
+import { existsSync, readFileSync } from 'fs';
+import { JWT } from 'google-auth-library';
+import { google, sheets_v4 } from 'googleapis';
+import { Credentials } from './interfaces';
+import { GoogleSheetsCmsSheet } from './sheets/GoogleSheetsCmsSheet';
+
+export interface GoogleSheetsCmsConfig extends PluginConfig {
+  caching?: boolean;
+  credentialsFile: string;
+  spreadsheetId?: string;
+  sheets: Record<string, GoogleSheetsCmsSheet>;
+}
+
+export class GoogleSheetsCms extends Plugin<GoogleSheetsCmsConfig> {
+  private jwt?: JWT;
+
+  getDefaultConfig(): GoogleSheetsCmsConfig {
```

Comment:
Should we maybe add a `translations` sheet to the default config? let's chat about this tomorrow